neurom/view/common.py

```diff
@@ -381,11 +381,8 @@ def plot_style(fig, ax, **kwargs):
         plt.show()  # pragma no cover
 
 
-def plot_title(fig, ax, **kwargs):
-
-    """
-    Function that defines the title options
-    of a matplotlib plot.
+def plot_title(_, ax, **kwargs):
```

Comment:
It makes sense. I think you are right to assume that it is not used in those functions because for >90% of the matplotlib operations the axes object is sufficient. However, in some cases we need to manipulate the figure in the output (that's why I was passing this as a argument). I would propose to keep it in the plot_style function but I think we could remove it from the rest especially if it is indeed not used.